Firely.Fhir.Packages.Tests/FhirPackageSourceTest.cs

```diff
@@ -133,5 +133,18 @@ public async Task TestGetArtifactByUri()
             pat.Should().NotBeNull();
             pat.Should().Contain("\"url\":\"http://hl7.org/fhir/StructureDefinition/Patient\"");
         }
+
+        [TestMethod]
+        public async Task TestFhirCorePackages()
+        {
+            var packageSource = FhirPackageSource.CreateCorePackageSource(new ModelInspector(FhirRelease.R5), FhirRelease.R5, "http://packages2.fhir.org/packages/");
+            var pat = await packageSource!.ResolveByCanonicalUriAsyncAsString("http://hl7.org/fhir/StructureDefinition/Patient");
+            pat.Should().NotBeNull();
```

Comment:
Can you add a test to test that you get a 5.0.0 version back of the StructureDefinition? (`fhirVersion`) That would make more sense to me (testing wise).